### DISCUSSION TITLE
Why not GCC support?

### DISCUSSION THREAD

**Discussion by widiba03304** [google/filament#7872] [General]
I found the Filament project very interesting, but I am having difficulty integrating it with other systems that are typically compiled with GNU C/C++ compilers.

Currently, the Filament project supports only Clang and libc++. Is there a specific reason for this choice? Are there any plans to support GNU compilers in the future?

Additionally, I am interested in building the project with Bazel. However, the project structure is complex, and it is challenging to create the necessary BUILD files. Is there any chance of adding Bazel support to this project?

Thank you for your time and consideration.

**Comment by romainguy** [comment]
We don't have any plans to support gcc and Bazel at this time.

**Reply by widiba03304** [reply]
Thanks for the quick reply. But I want to know what the specific reason for this. libc++ and lld seem not fully compatible with libstdc++ and gold. So, it would be great if there is an option for compiling this project from source with GCC to integrate with prebuilt binaries that are compiled with GCC (e.g., binaries installed with apt-get).

**Comment by widiba03304** [comment]
I understand gcc supports can have low priority now, but if there's a chance to make a PR for this, I would love to work on it.
If there was a serious issue that the maintainers decided to only support clang, I hope to know the context.

Related issues: #6149 #7386
